packages/rpc/test/clightning_rpc_test.dart

```diff
@@ -43,5 +43,26 @@ void main() {
       await client.simpleCall('delinvoice', params: params);
       client.close();
     });
+
+    test('Pay invoice method test', () async {
+      var client = RPCClient();
+      client.connect(rpcPath);
+      expect(client, isNotNull);
+      var params = <String, dynamic>{};
+      params['msatoshi'] = '100000msat';
+      params['label'] = 'from-dart';
+      params['description'] = 'This is a unit test';
+
+      var response = await client.simpleCall('invoice', params: params);
+      params = <String, dynamic>{};
+      params['bolt11'] = response["bolt11"];
+      await client.simpleCall('pay', params: params);
```

Comment:
if this test pass without any error this means that there is some mistake around,  we should have an exception